es6-classes/circle.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Circle */
+class Circle extends Shape {
+  constructor(radius) {
+    super(radius);
+    this.radius = radius;
+    this.area = Math.PI * radius ** 2;
+    this.perimeter = 2 * Math.PI * radius;
```

Comment:
You shouldn't be creating these properties manually. You should be utilizing `super` to generate these properties for you